app/src/processing/app/contrib/StatusPanel.java

```diff
@@ -235,9 +236,9 @@ public void update(ContributionPanel panel) {
       latestVersion = "Update";
     }
 
-    if(currentVersion != null){
-      currentVersion = "Version " + currentVersion;
-    }else{
+    if (currentVersion != null) {
+      currentVersion = "v" + currentVersion;
```

Comment:
Yes, but via `Language.interpolate()`, and make the string "%s available" so that it can be localized to languages that might need the equivalent of "available" in front.